cmd/buildah/bud.go

```diff
@@ -43,18 +43,15 @@ var (
 			Usage: "add global flags for the container runtime",
 		},
 		cli.StringFlag{
-			Name:  "format",
-			Usage: "`format` of the built image's manifest and metadata",
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
 		},
 		cli.StringSliceFlag{
 			Name:  "tag, t",
 			Usage: "`tag` to apply to the built image",
 		},
-		cli.StringSliceFlag{
```

Comment:
Don't we still need file?

Reply:
Yes we do.  Nice catch.  I wish this would be caught by the compiler.

Fixed